src/components/TeacherComponents/TermsAgreementForm/TermsAgreementForm.tsx

```diff
@@ -49,7 +50,11 @@ const TermsAgreementForm: FC<TermsAgreementFormProps> = ({
   handleEditDetailsCompletedClick = () => {},
   showPostAlbCopyright = true,
   copyrightYear,
+  hideDetails,
 }) => {
+  if (hideDetails) {
+    return null;
+  }
```

Comment:
Why not do `{!hideDetails && <TermsAgreementForm />}` in the consumer rather than adding a new prop?